bitnami/kafka/templates/statefulset.yaml

```
@@ -177,7 +177,7 @@ spec:
                   fieldPath: metadata.name
             - name: KAFKA_CFG_ZOOKEEPER_CONNECT
               {{- if .Values.zookeeper.enabled }}
-              value: {{ include "kafka.zookeeper.fullname" . | quote }}
+              value: "{{ printf "%s%s" (include "kafka.zookeeper.fullname" .) (tpl .Values.zookeeperChrootPath .) }}"
```

You don't need the quotes, use `quote` instead. Also, wouldn't it be necessary to add a forward slash between the hostname and the chroot path?
```suggestion
              value: {{ printf "%s/%s" (include "kafka.zookeeper.fullname" .) (tpl .Values.zookeeperChrootPath .) | quote }}
```

---

Quotes was changed but I don't think that necessary to add a forward slash. Because it creates unexpected behavior for path started without forward slash.